app/templates/components/reminder-item.hbs

```diff
@@ -0,0 +1,13 @@
+<div class='spec-reminder-item'>
+
+  <h3>{{reminder.title}}</h3>
+
+  {{#if reminder.date}}
```

Comment:
There should always be a date with a reminder so this conditional is unnecessary.